tests/scripts/command/nnpkg-test

```diff
@@ -14,7 +14,7 @@ difftool=${difftool:-"h5diff"}
 delete_dumped_on_failure=0
 
 usage() {
-  echo "Usage: $progname [options] nnpackage_test"
+  echo "Usage: $0 $(basename ${BASH_SOURCE[0]}) [options] nnpackage_test"
```

Comment:
@hseok-oh If you would like to inline `progname`, you can remove `progname=$(basename "${BASH_SOURCE[0]}")` at line 9.